src/Host.h

```diff
@@ -65,81 +65,192 @@ class Host : public QObject
     Host(int port, const QString& mHostName, const QString& login, const QString& pass, int host_id);
     ~Host();
 
-    QString            getName()                        { QMutexLocker locker(& mLock); return mHostName; }
```

Comment:
I'd like to see this section excluded from the formatting, given how we're not running the tool over entire files as a policy yet.